src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -15,7 +15,6 @@
 import seedu.address.logic.commands.EditCommand;
 import seedu.address.logic.commands.ExitCommand;
 import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.FindNricCommand;
```

Comment:
It is good that you have removed all traces of findnric command and hence avoided dead code